src/status_im/ui/screens/home/views.cljs

```diff
@@ -72,6 +73,11 @@
   (views/letsubs [home-items [:home-items]
                   show-welcome? [:get-in [:accounts/create :show-welcome?]]
                   view-id [:get :view-id]]
+    {:component-did-mount (fn []
```

Comment:
i'm not sure we can use `did-mount` because this component can be remounted, and also why did you decide to add `platform/ios?` ? i think better to create `fx` and use it in events flow, thanks